doc/controller-card_poweroff.md

```diff
@@ -0,0 +1,132 @@
+## Objective:
+
+This document introduces the concept of persistent shutdown of controller-cards in switches/routers with dual controller-cards, outlines rules to determine its use and proposes new telemetry counters to support the feature.
+
+
+## Possible use cases for the gNMI approach:
+
+
+
+1. Operator suspects faulty system behavior due to faulty hardware and wants to take it offline so they can revisit the issue in a maintenance window. But while this happens, the operator also wants to ensure that the card stays disabled across reboots so that the subject card does not take a primary role after reboot.
+2. It is possible that the faulty controller card is in a boot loop and the operator finds it best to disable the card to prevent undesired state. Expectations are also that the state persists even post a system reboot.
```

Comment:
This reads as an example of item 1.  Indent this?  Maybe add another example where a CPU is overheating if left powered on for more than a few minutes.

Reply:
Done, ptal!